src/ManifoldMuseum.jl

```diff
@@ -37,4 +46,20 @@ geodesic(M::Manifold, x, y, t) = exp(M, x, log(M, x, y), t)
 
 include("Sphere.jl")
 
+export Manifold
+export dimension,
+    distance,
+    dot,
+    exp,
```

Comment:
Sorry, I'm still new to Julia, but since we extended `Base.exp` here, do we need to export `exp`?

Reply:
Right, functions from `Base` don't have to be exported. I'll remove these exports.